modules/qna/src/views/full/FormModal.jsx

```diff
@@ -178,9 +201,29 @@ export default class FormModal extends Component {
 
   deleteAnswer = index => {
     const answers = this.state.item.answers
-    if (answers[index]) {
+    const answer = answers[index]
+    if (answer) {
       answers.splice(index, 1)
       this.changeItemProperty('answers', answers)
+      //Will set as deleted if it is a contentElement and isn't used as another answer
+      if (answer.contentId && !answers.find((a, aIndex) => { console.log(a, aIndex); return a.contentId == answer.contentId })) {
+        this.setState({ deletedContentElements: [...this.state.deletedContentElements, answer.contentId] })
+      }
```

Comment:
Console log, strict check, and it checks the use of content elements here, but it doesn't check if the flow editor uses it or something else.